src/main/java/hw10/configuration.txt

```diff
@@ -0,0 +1,4 @@
+FILE- C:/hillel/hillel_homework/src/main/java/hw10/application.log
```

Comment:
Using an absolute path is always a bad idea, so I would suggest using a relative path and creating log files under your project root, just create a folder `logs`, so the path to your log files will look like `logs/application.log`